src/react-native/logger.js

```diff
@@ -1,11 +1,9 @@
-'use strict';
-
 /* eslint-disable no-console */
-var logger = {
+const logger = {
```

Comment:
👍

Reply:
We should make a whole var -> const/let migration after.